src/CHeB.cpp

```diff
@@ -1490,8 +1490,8 @@ STELLAR_TYPE CHeB::ResolveEnvelopeLoss(bool p_NoCheck) {
         m_Tau = ((m_Age - tHeIPrime) / tHePrime);                                   // Hurley et al. 2000, just after eq 81
         m_Age = m_Tau * HeMS::CalculateLifetimeOnPhase_Static(m_Mass0);             // Hurley et al. 2000, eq 76 and following discussion
 
-        CalculateTimescales(m_Mass0, m_Timescales);
         CalculateGBParams(m_Mass0, m_GBParams);
+        CalculateTimescales(m_Mass0, m_Timescales);
```

Comment:
This seems like a very dramatic change, what is the reasoning here?

Reply:
Timescales are calculated using GBParams.  All other calls to CalculateTimescales() follow a call to CalculateGBParams(). I think this one was mistakenly reversed.

Reply:
Has it been reversed this whole time? I would expect that to change functionality for CHeB stars. 

I originally thought this was related to the HeHG p and q constants (PR #744), which was complicated to resolve because of the order of the function calls, or something like that.